build/commands/lib/config.js

```diff
@@ -285,13 +322,16 @@ Config.prototype.buildArgs = function () {
     enable_cdm_host_verification: this.enableCDMHostVerification(),
     enable_pseudolocales: this.enable_pseudolocales,
     skip_signing: !this.shouldSign(),
-    chrome_pgo_phase: this.chromePgoPhase,
     sparkle_dsa_private_key_file: this.sparkleDSAPrivateKeyFile,
     sparkle_eddsa_private_key: this.sparkleEdDSAPrivateKey,
     sparkle_eddsa_public_key: this.sparkleEdDSAPublicKey,
     ...this.extraGnArgs,
   }
 
+  if (args.is_official_build && !this.usePGO()) {
```

Comment:
if usePGO is dependent on official build then the check should be in `usePGO`. Spreading the logic around makes this more confusing imo

Reply:
Ok, removed `is_official_build`. The idea was not to set `chrome_pgo_phase = 0` when it's not necessary.